README.md

```diff
@@ -10,7 +10,7 @@ Conduit is an ultralight service mesh for Kubernetes from the makers of [Linkerd
 features a native proxy, written in [Rust][rust], that boasts the performance of C without
 all the heartbleed.
 
-Conduit is **experimental**. Currently, it _only supports HTTP/2_ and is especially
+Conduit is **experimental**. It _supports TCP, HTTP/1, and HTTP/2_ and is especially
```

Comment:
What about something like: Conduit is **experimental**. It is capable of proxying all TCP traffic, and reporting top-line metrics (success rates, latencies, etc) for all HTTP, HTTP/2, and gRPC traffic. It currently does not work with websockets or with HTTP tunneling--see the `--skip-outbound-ports` flag for how to exclude these types of traffic.